website/blog/2020-01-10-cobalt.md

```diff
@@ -0,0 +1,718 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.0 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+After a long while of silence and with loads of new features we are excited to
+announce the release of Metals v0.8.0 codename `Cobalt`. This release includes a
+huge number of new features and improvements.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
```

Comment:
Add Lunatech?

Reply:
I was thinking about it and wanted to ask @ckipp01 , but totally forgot :sweat: 

@ckipp01 what do you think about it?

Reply:
I'm all for it if you guys are willing. I thought about it when I read it, but didn't want to be pushy about it. However, I know Lunatech would really appreciate it.

Reply:
Added, is it alright?

Reply:
Yup, that's great!